packages/dashboard/src/settings.ts

```diff
@@ -29,5 +35,6 @@ export function defaultSettings(): Settings {
   return {
     trajectoryAnimation: TrajectoryAnimation.Follow,
     themeMode: ThemeMode.Light,
+    useTheme: UseTheme.False,
```

Comment:
This is give users the option to switch back to the original material theme we are using. Since this feature is optional and not really complete yet, I thought it would be good to let users have this option.

Reply:
The problem is that even with `useTheme === false`, there are still some changes in this PR to the material theme. It is also confusing to have such an option, better to just make light theme be the material default and dark be rmf-dark in that case. Or have the theme be a dropdown with 3 options.